phpBB/phpbb/template/twig/loader.php

```diff
@@ -23,6 +23,16 @@ class loader extends \Twig_Loader_Filesystem
 	protected $safe_directories = array();
 
 	/**
+	 * Constructor
+	 *
+	 * @param string|array	$paths
+	 */
+	public function __construct($paths = array())
+	{
+		parent::__construct($paths, filesystem_helper::realpath(dirname(__FILE__)));
```

Comment:
Just use `parent::__construct($paths, __DIR__);`?

Reply:
Looks like working as well. If so, it could be applied to 3.2 too.

Reply:
Yes, is exactly the same.
https://secure.php.net/manual/en/language.constants.predefined.php

Reply:
Exactly my point, what you did is pointless.

Reply:
I haven't done this, i just restore a code that was accidentally removed

Reply:
https://pastebin.com/ANeWhryy

Reply:
`__DIR__` is better because it does not require a function call at execution time, so it's faster.

Reply:
https://tracker.phpbb.com/browse/PHPBB3-15392